packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown-styles.spec.ts

```diff
@@ -59,39 +59,46 @@ describe('getFilterStyles()', () => {
 
 describe('getListStyles()', () => {
   it.each<Parameters<typeof getListStyles>>([
-    ['down', 'light'],
-    ['down', 'light'],
-    ['down', 'dark'],
-    ['down', 'dark'],
-    ['up', 'light'],
-    ['up', 'light'],
-    ['up', 'dark'],
-    ['up', 'dark'],
-  ])('should return correct css for direction: %s, isOpen: %s and theme: %s', (direction, theme) => {
-    expect(getCss(getListStyles(direction, theme))).toMatchSnapshot();
-  });
+    ['down', 'light', false],
+    ['down', 'light', false],
+    ['down', 'light', true],
+    ['down', 'dark', false],
+    ['down', 'dark', false],
+    ['up', 'light', false],
+    ['up', 'light', true],
+    ['up', 'light', false],
+    ['up', 'dark', false],
+    ['up', 'dark', false],
+  ])(
+    'should return correct css for direction: %s, isOpen: %s and theme: %s, isNativePopover: %s',
```

Comment:
3 parameters are defined in array but 4 are mentioned here so besides mixup the last one will always be undefined.